mirdata/phenicx_anechoic.py

```diff
@@ -0,0 +1,654 @@
+    """
+    if data_home is None:
+        data_home = utils.get_default_dataset_path(DATASET_DIR)
+
+    data = {}
+    for key in DATA.index.keys():
+        data[key] = Track(key, data_home=data_home)
+    return data
+
+
+# -- Write any necessary loader functions for loading the dataset's data
+def load_score(score_paths):
+    """
+    Args:
+        score_paths (str) or list[str]: list of txt score files containing start_time, end_time, note
+    Returns:
+        utils.EventData: score as EventData tuple
+    """
+
+    if isinstance(score_paths, str):
+        score_paths = list(score_paths)
+    assert isinstance(
+        score_paths, list
+    ), "score_paths should be either string or list of strings"
+
+    start_times = []
+    end_times = []
+    score = []
+    for path in score_paths:
+        if not os.path.exists(path):
+            raise IOError("path {} does not exist".format(path))
+
+        #### read start, end times
+        times = np.loadtxt(path, delimiter=",", usecols=[0, 1], dtype=np.float)
+        start_times.append(times[:, 0])
+        end_times.append(times[:, 1])
+
+        #### read notes as string
+        with open(path) as f:
+            content = f.readlines()
+            sc = np.array([line.split(',')[2].strip('\n') for line in content])
+            score.append(sc)
+
+    start_times = np.concatenate(start_times)
+    end_times = np.concatenate(end_times)
+    score = np.concatenate(score)
+
+    # sort on the start time
+    ind = np.argsort(start_times, axis=0)
+    start_times = np.take_along_axis(start_times, ind, axis=0)
+    end_times = np.take_along_axis(end_times, ind, axis=0)
+    score = np.take_along_axis(score, ind, axis=0)
+
+    data = utils.EventData(start_times, end_times, score)
+    return data
+
+
+def cite():
+    """Print the reference"""
+
+    cite_data = """
+=========== MLA ===========
+Miron, Marius, et al. "Score-informed source separation for multichannel orchestral recordings." Journal of Electrical and Computer Engineering 2016 (2016).
+
+Pätynen, Jukka, Ville Pulkki, and Tapio Lokki. "Anechoic recording system for symphony orchestra." Acta Acustica united with Acustica 94.6 (2008): 856-865.
+========== Bibtex ==========
+@article{miron2016score,
+  title={Score-informed source separation for multichannel orchestral recordings},
+  author={Miron, Marius and Carabias-Orti, Julio J and Bosch, Juan J and G{\'o}mez, Emilia and Janer, Jordi},
+  journal={Journal of Electrical and Computer Engineering},
+  volume={2016},
+  year={2016},
+  publisher={Hindawi}
+}
+
+@article{patynen2008anechoic,
+  title={Anechoic recording system for symphony orchestra},
+  author={P{\"a}tynen, Jukka and Pulkki, Ville and Lokki, Tapio},
+  journal={Acta Acustica united with Acustica},
+  volume={94},
+  number={6},
+  pages={856--865},
+  year={2008},
+  publisher={S. Hirzel Verlag}
+}
+
+"""
+    print(cite_data)
+
+
+##########################################
```

Comment:
hi Rachel
thank you for replying!

I want to be able to use these files for source separation and I think a simple sum of audios is not enough. 
what I need:
        sources (OrderedDict): dictionary comprising Source objects, one source represents one audio file - we have multiple audio files per Track
        targets (OrderedDict): dictionary comprising Target objects, one Target is a linear mix of multiple sources - think of these as groups. for instance voice, drums, bass, other are groups but so is accompaniment as drums+bass+other 
        mix (Target): a Target object, a linear mix of all sources - this is what you probably suggested as a simple sum of sources

I could have done it in a similar way to musdb_sources but that requires a similar implementation of Target and Source after calling the mirdata object. the thing we want to avoid is non-standard ways of working with a dataset. so that's why I proposed these objects: Target and Source. so everyone mixes tracks in the same way
This code is derived from musdb. I think it's one of the best examples to implement multi-tracks for a given Track (if you look at their repo, the Multi-Track object is the same as our Track). 

let's see where the discussion goes. I am open to some changes. btw, there is a possibility that each Source is multi-channel (stereo or more). in the original experiments phenicx-anechoic was multichannel having 6 mics and a source .wav file in each mic. I didn't implement this because it was already too much. but once this is solve I will start working on it.